app/helpers/shift_commerce/cache_helper.rb

```diff
@@ -21,19 +22,19 @@ def menus_cache(reference, banner_reference = nil, dependent_reference = nil, op
 
```

Comment:
A thought, can we read the cache version from the constant rather than it being passed in constantly.
```suggestion
      menu_cache_version = Menus.menu_cache_version
```